react-app/src/providers/QueryClientProvider.tsx

```diff
@@ -37,9 +48,19 @@ const QueryClientProvider: React.FC<QueryClientProviderProps> = (props) => {
       });
   }, [chainInfo]);
 
+  useEffect(() => {
+    newDesmosQueryClient(desmosRpc)
+      .then((client) => {
+        setDesmosQueryClient(client);
+      })
+      .catch((err) => {
+        console.log("Error creating desmos query client", err);
+      });
+  }, [desmosRpc]);
+
   return (
     <QueryClientContext.Provider value={value}>
-      {!!queryClient && children}
+      {!!(queryClient && desmosQueryClient) && children}
```

Comment:
can you help to change this whole thing to `!= null`? 🙏

Reply:
okay changed, do you mean like this?

```ts
{(queryClient && desmosQueryClient) != null && children}
```